client/flutter/assets/content_bundles/your_questions_answered.fr_FR.yaml

```diff
@@ -0,0 +1,221 @@
+
+        <p>💉 <b>Existe-t-il un vaccin, un médicament ou un traitement contre la COVID-19? </b></p>
+
+        <p>Pas encore. Jusqu&#39;à présent, il n&#39;existe aucun vaccin et aucun médicament antiviral spécifique pour prévenir ou guérir la COVID-2019. Cependant, les personnes concernées devraient recevoir des soins pour soulager les symptômes. Les personnes gravement malades doivent être hospitalisées. La plupart des patients se rétablissent grâce aux soins de soutien.</p>
+
+        <p>Des vaccins possibles et certains traitements médicamenteux spécifiques sont à l&#39;étude. Ils sont testés dans le cadre d&#39;essais cliniques. L&#39;OMS coordonne ses efforts pour mettre au point des vaccins et des médicaments pour prévenir et traiter la COVID-19.</p>
+
+        <p>La meilleure façon de se protéger et de protéger les autres contre le COVID-19 consiste à se laver fréquemment les mains, à se couvrir la bouche avec le pli du coude ou avec un mouchoir, et à se tenir à une distance d&#39;au moins 1 mètre (3 pieds) de toute personne qui tousse ou qui éternue. </p>
+    -
+      title_html: |
+        Dois-je porter un masque pour me protéger ?
+      body_html: |
+        <p>Ne portez un masque que si vous êtes malade avec des symptômes de COVID-19 (en particulier la toux) ou si vous prenez soin d&#39;une personne susceptible d&#39;être atteinte du COVID-19. Les masques jetables sont à usage unique. Si vous portez un masque alors que vous n’êtes pas malade ou que vous ne vous occupez pas de quelqu’un qui est malade, c’est du gaspillage. Comme il y a une pénurie mondiale de masques, l’OMS conseille de les utiliser avec parcimonie.</p>
+
+        <p>L&#39;OMS recommande de faire un usage rationnel des masques médicaux afin d&#39;éviter le gaspillage de ressources précieuses et l&#39;utilisation abusive de masques.</p>
+
+        <p>La meilleure façon de se protéger et de protéger les autres contre le COVID-19 consiste à se laver fréquemment les mains, à se couvrir la bouche avec le pli du coude ou avec un mouchoir, et à se tenir à une distance d&#39;au moins 1 mètre (3 pieds) de toute personne qui tousse ou qui éternue. </p>
+
+        <p>🚮 <b>Comment mettre, utiliser, enlever et éliminer un masque ?</b></p>
+
+        <ol>
+          <li>Ne pas oublier que seuls les agents de santé, les personnes qui s’occupent de malades et les personnes qui présentent des symptômes respiratoires (fièvre et toux) doivent porter un masque.</li>
```

Comment:
This might be valid in france, however certain country force the use of mask in public for grocery shopping. Adding a "Conformez vous aux règles en vigueur." can be a nice addon

Reply:
To be reviewed by WHO in RC3